diff --git a/flake.nix b/flake.nix
index 97aa95d..cdf3622 100644
--- a/flake.nix
+++ b/flake.nix
@@ -63,31 +63,34 @@
       #         home-modules
       #       ];
       #     };
-      #
-      #   ginyu = let
-      #     username = "andrew";
-      #   in
-      #     nixpkgs.lib.nixosSystem {
-      #       system = "x86_64-linux";
-      #       specialArgs = {inherit inputs username;};
-      #       modules = [
-      #         ./hosts/xps9300/default.nix
-      #
-      #         home-manager.nixosModules.home-manager
-      #         {
-      #           home-manager.extraSpecialArgs = {inherit inputs username;};
-      #           home-manager.users."${username}" = {
-      #             imports = [
-      #               ./home/default.nix
-      #               ./home/wayland.nix
-      #             ];
-      #           };
-      #         }
-      #         home-modules
-      #         nixos-hardware.nixosModules.dell-xps-13-9360
-      #       ];
-      #     };
-      #
+
+      ginyu = let
+        # xps
+        username = "dragon";
+      in
+        nixpkgs.lib.nixosSystem {
+          system = "x86_64-linux";
+          specialArgs = {inherit inputs username;};
+          modules = [
+            ./hosts/ginyu/default.nix
+            home-manager.nixosModules.home-manager
+            ./modules
+
+            {
+              home-manager.extraSpecialArgs = {inherit inputs username;};
+              home-manager.users."${username}" = {
+                imports = [
+                  ./home
+                  ./hosts/ginyu/home.nix
+                  ./hm-modules
+                ];
+              };
+            }
+            home-modules
+            nixos-hardware.nixosModules.dell-xps-13-9360
+          ];
+        };
+
       dende = let
         username = "dragon";
       in
diff --git a/hm-modules/niri.nix b/hm-modules/niri.nix
index 5045cbf..bbb80a1 100644
--- a/hm-modules/niri.nix
+++ b/hm-modules/niri.nix
@@ -34,8 +34,9 @@ in {
     packages = mkOption {
       description = "extra packages for Niri to work nicely";
       type = listOf package;
-      default = [
-        pkgs.xwayland-satellite
+      default = with pkgs; [
+        xwayland-satellite
+        xorg.libXcursor
       ];
     };
   };
diff --git a/home/default.nix b/home/default.nix
index cfe3753..732791a 100644
--- a/home/default.nix
+++ b/home/default.nix
@@ -12,7 +12,7 @@ in {
   home.homeDirectory = lib.mkDefault "/home/${username}";

   imports = [
-    ./programs/anki.nix
+    #./programs/anki.nix
     ./programs/atuin.nix
     ./programs/firefox.nix
     ./programs/git.nix
@@ -148,7 +148,7 @@ in {

       (callPackage ../pkgs/rbonsai.nix {})
       #(callPackage ../pkgs/cider-wrapper.nix {})
-      (callPackage ../pkgs/ziti/bin.nix {})
+      #(callPackage ../pkgs/ziti/bin.nix {})
     ])
     ++ [
       neovim-flake
diff --git a/hosts/dende/home.nix b/hosts/dende/home.nix
index 7ab8478..69260b8 100644
--- a/hosts/dende/home.nix
+++ b/hosts/dende/home.nix
@@ -128,6 +128,5 @@ in {
     niri.enable = true;
   };
   zah-hm.programs.dunst.enable = true;
-
   services.blueman-applet.enable = true;
 }
diff --git a/hosts/ginyu/default.nix b/hosts/ginyu/default.nix
new file mode 100644
index 0000000..e9de374
--- /dev/null
+++ b/hosts/ginyu/default.nix
@@ -0,0 +1,64 @@
+{
+  pkgs,
+  username,
+  ...
+}: {
+  imports = [
+    ../common.nix
+    ../common-linux.nix
+    ./hardware-configuration.nix
+
+    ../../services/bluetooth.nix
+    ../../services/fcitx.nix
+    ../../services/keyd.nix
+
+    ./power.nix
+    ./services.nix
+
+    ./nix.nix
+    ./remotebuild.nix
+    #./wireguard-client.nix
+    ./unfree.nix
+  ];
+
+  networking.hostName = "ginyu";
+  time.timeZone = "America/New_York";
+  users.users."${username}".extraGroups = ["docker" "input" "wheel" "networkmanager"];
+  system.stateVersion = "25.11";
+
+  programs.zsh.enable = true;
+  programs.zsh.shellAliases = {
+    switch = "sudo nixos-rebuild switch --flake ~/nix#ginyu";
+  };
+  programs.niri.enable = true;
+
+  programs.gnupg.agent = {
+    enable = true;
+    pinentryPackage = pkgs.pinentry-curses;
+  };
+
+  hardware.opengl = {
+    enable = true;
+    extraPackages = with pkgs; [
+      (vaapiIntel.overrideAttrs (prev: {
+        meta.priority = 1;
+      }))
+    ];
+  };
+
+  virtualisation = {
+    docker.enable = true;
+    docker.autoPrune.enable = true;
+
+    containerd.enable = true;
+  };
+  #boot.binfmt.emulatedSystems = [ "aarch64-linux" ];
+
+  # 'sg' for dvd drives w/ encrypted content
+  #boot.kernelModules = ["sg"];
+
+  #programs.neovim-flake = {
+  #  enable = true;
+  #  background = "light";
+  #};
+}
diff --git a/hosts/xps9300/hardware-configuration.nix b/hosts/ginyu/hardware-configuration.nix
similarity index 100%
rename from hosts/xps9300/hardware-configuration.nix
rename to hosts/ginyu/hardware-configuration.nix
diff --git a/hosts/ginyu/home.nix b/hosts/ginyu/home.nix
new file mode 100644
index 0000000..57f28c7
--- /dev/null
+++ b/hosts/ginyu/home.nix
@@ -0,0 +1,28 @@
+{pkgs, ...}: {
+  imports = [
+    ../../home/programs/ghostty.nix
+  ];
+
+  home.packages = with pkgs; [
+    xorg.libXcursor
+  ];
+
+  home.pointerCursor = {
+    package = pkgs.bibata-cursors;
+    name = "Bibata-Modern-Classic";
+    size = 20;
+    gtk.enable = true;
+    x11.enable = true;
+  };
+
+  zah-hm.graphics = {
+    x11.enable = false;
+    i3.enable = false;
+    i3status-rust.enable = false;
+
+    wayland.enable = true;
+    waybar.enable = true;
+    niri.enable = true;
+  };
+  zah-hm.programs.dunst.enable = true;
+}
diff --git a/hosts/xps9300/nix.nix b/hosts/ginyu/nix.nix
similarity index 100%
rename from hosts/xps9300/nix.nix
rename to hosts/ginyu/nix.nix
diff --git a/hosts/xps9300/power.nix b/hosts/ginyu/power.nix
similarity index 100%
rename from hosts/xps9300/power.nix
rename to hosts/ginyu/power.nix
diff --git a/hosts/xps9300/remotebuild.nix b/hosts/ginyu/remotebuild.nix
similarity index 100%
rename from hosts/xps9300/remotebuild.nix
rename to hosts/ginyu/remotebuild.nix
diff --git a/hosts/ginyu/services.nix b/hosts/ginyu/services.nix
new file mode 100644
index 0000000..d86e4eb
--- /dev/null
+++ b/hosts/ginyu/services.nix
@@ -0,0 +1,18 @@
+{...}: {
+  services.sshd.enable = true;
+  services.tailscale.enable = false;
+
+  services.xserver = {
+    enable = true;
+    videoDrivers = ["amd"];
+
+    xkb.layout = "us";
+    xkb.variant = "";
+  };
+
+  userExtraServices = {
+    ghostty-daemon.enable = true;
+  };
+
+  programs.xwayland.enable = true;
+}
diff --git a/hosts/xps9300/unfree.nix b/hosts/ginyu/unfree.nix
similarity index 100%
rename from hosts/xps9300/unfree.nix
rename to hosts/ginyu/unfree.nix
diff --git a/hosts/xps9300/wireguard-client.nix b/hosts/ginyu/wireguard-client.nix
similarity index 100%
rename from hosts/xps9300/wireguard-client.nix
rename to hosts/ginyu/wireguard-client.nix
diff --git a/hosts/xps9300/default.nix b/hosts/xps9300/default.nix
deleted file mode 100644
index 0585a3d..0000000
--- a/hosts/xps9300/default.nix
+++ /dev/null
@@ -1,104 +0,0 @@
-{
-  pkgs,
-  username,
-  ...
-}: {
-  imports = [
-    ../common.nix
-    ../common-linux.nix
-    ./hardware-configuration.nix
-
-    ../../services/bluetooth.nix
-    ../../services/fcitx.nix
-    ../../services/keyd.nix
-
-    ./power.nix
-    ./services.nix
-
-    ./nix.nix
-    ./remotebuild.nix
-    #./wireguard-client.nix
-    ./unfree.nix
-  ];
-
-  networking.hostName = "xps9300";
-  time.timeZone = "America/New_York";
-  users.users."${username}".extraGroups = ["docker" "input"];
-  system.stateVersion = "24.05";
-
-  # for work
-  services.hydra = {
-    enable = false;
-    hydraURL = "http://localhost:33000";
-    notificationSender = "hydra@localhost";
-    buildMachinesFiles = [];
-    useSubstitutes = true;
-  };
-
-  # ps5 increase polling rate - but does it work with bluetooth ?
-  #services.udev.extraRules = lib.concatStrings [
-  #  # increase polling rate to 1ms for ps5 dualsense
-  #  ''
-  #    ACTION=="bind", SUBSYSTEM=="hid", DRIVER=="sony", KERNEL=="*054C:0CE6*", ATTR{bt_poll_interval}="1"
-  #  ''
-  #];
-
-  #fileSystems."/mnt/cd1" = {
-  #  device = "/dev/sr0";
-  #  fsType = "auto";
-  #  options = [
-  #    "ro"
-  #    "user"
-  #    "noauto"
-  #    "unhide"
-  #  ];
-  #};
-
-  security.pki.certificateFiles = [
-    /home/${username}/work/platform/nix/local-proxy/pki/docker-registry.arpa.crt
-    /home/${username}/work/platform/nix/local-proxy/pki/attic.arpa.crt
-  ];
-
-  programs.sway = {
-    enable = true;
-    wrapperFeatures.gtk = true;
-    extraPackages = with pkgs; [
-      bibata-cursors
-    ];
-  };
-
-  programs.zsh.enable = true;
-  programs.zsh.shellAliases = {
-    switch = "sudo nixos-rebuild switch --flake ~/nix#xps9300";
-  };
-
-  programs.gnupg.agent = {
-    enable = true;
-    pinentryPackage = pkgs.pinentry-curses;
-  };
-
-  hardware.opengl = {
-    enable = true;
-    extraPackages = with pkgs; [
-      (vaapiIntel.overrideAttrs (prev: {
-        meta.priority = 1;
-      }))
-    ];
-  };
-
-  virtualisation = {
-    docker.enable = true;
-    docker.autoPrune.enable = true;
-
-    containerd.enable = true;
-  };
-  #boot.binfmt.emulatedSystems = [ "aarch64-linux" ];
-
-  # 'sg' for dvd drives w/ encrypted content
-  #boot.kernelModules = ["sg"];
-
-  #programs.neovim-flake = {
-  #  enable = true;
-  #  background = "light";
-  #};
-}
diff --git a/hosts/xps9300/services.nix b/hosts/xps9300/services.nix
deleted file mode 100644
index c95a608..0000000
--- a/hosts/xps9300/services.nix
+++ /dev/null
@@ -1,4 +0,0 @@
-{ ... }: {
-  services.sshd.enable = true;
-  services.tailscale.enable = false;
-}
